modules/distribution-zones/src/test/java/org/apache/ignite/internal/distributionzones/DistributionZoneManagerScaleUpTest.java

```diff
@@ -114,7 +114,13 @@ void testDataNodesPropagationAfterScaleUpTriggered() throws Exception {
 
         assertDataNodesForZone(1, clusterNodes2.stream().map(ClusterNode::name).collect(Collectors.toSet()), keyValueStorage);
 
-        assertZoneScaleUpChangeTriggerKey(1, 1, keyValueStorage);
+        topology.putNode(NODE_3);
+
+        Set<LogicalNode> clusterNodes3 = Set.of(NODE_1, NODE_2, NODE_3);
+
+        assertLogicalTopology(clusterNodes3, keyValueStorage);
+
+        assertDataNodesForZone(1, clusterNodes3.stream().map(ClusterNode::name).collect(Collectors.toSet()), keyValueStorage);
```

Comment:
It worth to mention that

- It's better to have `private static final int ZONE_1_ID = 1;` instead of 1.
- Almost everywhere were we assert assertDataNodesForZone for zone1 we may also assert data nodes for the default one, I'd rather add following method to `DistributionZonesTestUtil`
```
    public static void assertDataNodesForZones(
            @Nullable Set<String> clusterNodes,
            KeyValueStorage keyValueStorage,
            int... zoneIds
    ) throws InterruptedException {
        for (int zoneId : zoneIds) {
            assertDataNodesForZone(zoneId, clusterNodes, keyValueStorage);
        }
    }
```

Reply:
Fixed.